README.md

```diff
@@ -149,7 +149,7 @@ SchemaPlus::ForeignKeys is tested on:
 
 ## History
 
-* 1.0.0 - Compatibility with ActiveRecord 5.0. Removed support for 4.x
+* 1.0.0 - Compatibility with ActiveRecord 5.0.
```

Comment:
Can you look at the latest commit? It was a typo. I fixed it to 0.1.7, but still let this commit with 1.0.0 slip by before that.